hack/ci/Vagrantfile-flatcar

```diff
@@ -0,0 +1,118 @@
+      CNI_VERSION="v1.0.1"
+      RELEASE_VERSION="v0.4.0"
+      DOWNLOAD_DIR=/opt/bin
+      RELEASE="$(curl -sSL https://dl.k8s.io/release/stable.txt)"
+
+      mkdir -p $DOWNLOAD_DIR
+      mkdir -p /opt/cni/bin
+      mkdir -p /etc/systemd/system/kubelet.service.d
+
+      # Install tools
+      curl -sSL -o $DOWNLOAD_DIR/jq "https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64"
+      chmod +x $DOWNLOAD_DIR/jq
+      curl -sSL -o $DOWNLOAD_DIR/flatcar_developer_container.bin.bz2 "https://stable.release.flatcar-linux.net/amd64-usr/current/flatcar_developer_container.bin.bz2"
+      bzcat $DOWNLOAD_DIR/flatcar_developer_container.bin.bz2 > $DOWNLOAD_DIR/flatcar_developer_container.bin
+
+      curl -sSL "https://github.com/containernetworking/plugins/releases/download/${CNI_VERSION}/cni-plugins-linux-amd64-${CNI_VERSION}.tgz" | tar -C /opt/cni/bin -xz
+      curl -sSL "https://raw.githubusercontent.com/kubernetes/release/${RELEASE_VERSION}/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service" | sed "s:/usr/bin:${DOWNLOAD_DIR}:g" | tee /etc/systemd/system/kubelet.service
+      curl -sSL "https://raw.githubusercontent.com/kubernetes/release/${RELEASE_VERSION}/cmd/kubepkg/templates/latest/deb/kubeadm/10-kubeadm.conf" | sed "s:/usr/bin:${DOWNLOAD_DIR}:g" | tee /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+      curl -sSL --remote-name-all https://storage.googleapis.com/kubernetes-release/release/${RELEASE}/bin/linux/amd64/{kubeadm,kubelet,kubectl}
+
+      chmod +x {kubeadm,kubelet,kubectl}
+      mv {kubeadm,kubelet,kubectl} $DOWNLOAD_DIR/
+
+      # Disable kernel print rate limiting for syslog messaging
+      sysctl -w kernel.printk_ratelimit=0
+      sysctl -w kernel.printk_ratelimit_burst=0
+
+      # Import the test image for security profiles operator 
+      docker load -i /vagrant/image.tar
+
+      # Install cri-o
+      # /vagrant/hack/ci/install-cri-o.sh
+
+      # Disable IPv6 in bridge plugin config
+      # CNI_CONFIG=/etc/cni/net.d/10-crio-bridge.conf
+      # jq 'del(.ipam.routes[1], .ipam.ranges[1])' $CNI_CONFIG | sponge $CNI_CONFIG
+
+      systemctl enable --now kubelet
+      systemctl status kubelet
+
+      # Setup cluster
+      IP=`ip route get 1.2.3.4 | cut -d ' ' -f7 | tr -d '[:space:]'`
+      NODENAME=$(hostname -s)
+      swapoff -a
+      sysctl -w net.ipv4.ip_forward=1
+      kubeadm config images pull
+      kubeadm init --config /vagrant/hack/ci/kubeadm-config-flatcar.yaml --node-name $NODENAME
+
+      # Setup kubectl
+      mkdir -p $HOME/.kube
+      cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+      chown -R core:core $HOME/.kube
+      export KUBECONFIG=/etc/kubernetes/admin.conf
+
+      # Configure cluster
+      kubectl taint nodes --all node-role.kubernetes.io/master-
+
+      # Install CNI
+      kubectl create -f https://docs.projectcalico.org/manifests/tigera-operator.yaml
+      kubectl apply -f /vagrant/hack/ci/flatcar-cni-plugin.yaml
```

Comment:
Is there a reason why Calico is needed? Could we simply use the `bridge` SNI plugin like the other tests do?

Reply:
Somehow the cluster was not becoming ready without Calico. This is why I enabled it.